backend/api/tests/test_permissions.py

```
@@ -0,0 +1,253 @@
+    IsAdmin, 
+    IsAdminOrReadOnly, 
+    IsOwnerOrAdmin, 
+    IsCompanyOrAdmin
+)
+from api.models import Video
+
+User = get_user_model()
+
+class DummyView(APIView):
+    """A dummy view for testing permissions."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)